crowbar_framework/app/assets/javascripts/installer.js

```diff
@@ -0,0 +1,69 @@
+$(document).ready(function() {
+  $("body.installer").each(function() {
+    function statusCheck() {
+      $.ajax({
+        type: "GET",
+        dataType: "json",
+        url: Routes.installer_status_path(),
```

Comment:
Identifier 'installer_status_path' is not in camel case.<br>'Routes' is not defined.